python/src/aiconfig/editor/server/server.py

```diff
@@ -457,3 +457,30 @@ def set_description() -> FlaskResponse:
     operation = make_op_run_method(MethodName("set_description"))
     operation_args: Result[OpArgs, str] = result.Ok(OpArgs({"description": description}))
     return run_aiconfig_operation_with_op_args(aiconfig, "set_description", operation, operation_args)
+
+@app.route("/api/clear_outputs", methods=["POST"])
+def clear_outputs() -> FlaskResponse:
+    """
+    Clears all outputs in the server state's AIConfig.
+    """
+    state = get_server_state(app)
+    aiconfig = state.aiconfig
+
+    if aiconfig is None:
+        LOGGER.info("No AIConfig in memory, can't run clear outputs.")
+        return HttpResponseWithAIConfig(
+            message="No AIConfig in memory, can't run clear outputs.",
+            code=400,
+            aiconfig=None,
+        ).to_flask_format()
+
+    for prompt in aiconfig.prompts:
+        prompt_name = prompt.name
+        # fn name `delete_output`` is misleading. TODO: Rename to `delete_outputs`` in AIConfig API
+        aiconfig.delete_output(prompt_name)
+
+
+    return HttpResponseWithAIConfig(
```

Comment:
Jonathan already built `run_aiconfig_operation_with_op_args()`, let's use that

Reply:
I think run_aiconfig_operation_with_op_args() is not quite general enough.

 This is fine, but it turns out this should be easy enough like this:
 
 ```
 def _op(aiconfig_runtime: AIConfigRuntime, _op_args: OpArgs) -> Result[None, str]:
        for prompt in aiconfig_runtime.prompts:
            prompt_name = prompt.name
            # fn name `delete_output`` is misleading. TODO: Rename to `delete_outputs`` in AIConfig API
            aiconfig_runtime.delete_output(prompt_name)
        return Ok(None)
    signature: dict[str, Type[Any]] = {}
    return run_aiconfig_operation_with_request_json(aiconfig, request_json, f"method_{method_name}", _op, signature)
 ```

Reply:
updated. Thanks @jonathanlastmileai for the help, used `run_aiconfig_operation_with_request_json`.